src/messaging/ExchangeMgr.h

```diff
@@ -249,6 +255,9 @@ class DLL_EXPORT ExchangeManager : public SecureSessionMgrDelegate, public Trans
 
     Transport::AdminId mAdminId = 0;
 
+    // By default allocate the application transport. This is the most commonly used and secure transport.
+    ApplicationExchangeTransport mDefaultExchangeTransport;
```

Comment:
SecureSessionMgr * mSessionMgr; is stored both ExchangeMgr and ApplicationExchangeTransport, it is duplicated

Reply:
I'll update the code to remove it from ExchangeMgr, since instance of ApplicationExchangeTransport is contained in ExchangeMgr itself.